smoke-tests/spec/psp_spec.rb

```diff
@@ -0,0 +1,106 @@
+require "spec_helper"
+
+# Pod security policies (PSP) is a cluster-level resource that
+# controls security sensitive aspects of the pod specification.
+# This spec confirms the Cloud Platform psp's are operational
+# within its cluster.
+describe "pod security policies" do
+  let(:namespace) { "integrationtest-psp-#{readable_timestamp}" }
+  let(:pods) { get_running_pods(namespace)}
+
+  # Confirms the main psp's currently exist
+  specify "has the expected policies" do
+    names = get_psp.map { |set| set.dig("metadata", "name") }.sort
+
+    expected = [
+      "privileged",
+      "restricted",
+      "kube-system"
+    ]
+    expect(names).to include(*expected)
+  end
+
+  # Runs a privilege namespace and confirms both privilege and
+  # non-privilege containers can run. 
+  context "when namespace is privileged" do
+    before do
+      create_namespace(namespace)
+      make_namespace_privileged(namespace)
+    end
+
+    after do
+      delete_namespace(namespace)
+      delete_clusterrolebinding(namespace)
+    end
+
+    it "privileged containers run" do
```

Comment:
For the names of tests, generally you either want to use "it" to describe a situation, like 

> it "allows root containers to run" do ...

Or, if that produces names that are too clunky, you might use "specify" like this;

> specify "containers run as root" do ....

Reply:
Agreed. Changed the names of all descriptions.